cmake/sysbuild/bootconf.cmake

```
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+function(setup_bootconf_data)
+  if(SB_CONFIG_PARTITION_MANAGER)
+    add_custom_target(bootconf_target
+      DEPENDS ${CMAKE_BINARY_DIR}/bootconf.hex
+    )
+
+    add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/bootconf.hex
+      COMMAND ${Python3_EXECUTABLE}
+        ${ZEPHYR_NRF_MODULE_DIR}/scripts/reglock.py
+        -o ${CMAKE_BINARY_DIR}/bootconf.hex
+        -s $<TARGET_PROPERTY:partition_manager,PM_B0_SIZE>
```

short arg names are very convenient on command line, but inside CMake code like this, then using long names are more self-explanatory.
```suggestion
        --ouput ${CMAKE_BINARY_DIR}/bootconf.hex
        --size $<TARGET_PROPERTY:partition_manager,PM_B0_SIZE>
```

---

Fixed